pytorch_forecasting/metrics.py

```diff
@@ -835,3 +835,37 @@ def rescale_parameters(
         loc = parameters[..., 0] * target_scale[..., 1].unsqueeze(-1) + target_scale[..., 0].unsqueeze(-1)
 
         return torch.stack([loc, scale], dim=-1)
+
+
+class BetaDistributionLoss(DistributionLoss):
+    """
+    Beta distribution loss for unit interval data.
+
+    Requirements for original target normalizer:
+        * coerced to be positive
+        * not centered normalization (only rescaled)
+        * normalized target not in log space
+    """
+    distribution_class = distributions.Beta
+    distribution_arguments = ["mean", "shape"]
+
+    def map_x_to_distribution(self, x: torch.Tensor) -> distributions.Beta:
+        mean = x[..., 0]
+        shape = x[..., 1]
+        return self.distribution_class(concentration0=(1 - mean) * shape, concentration1=mean * shape)
+
+    def rescale_parameters(
+        self, parameters: torch.Tensor, target_scale: torch.Tensor, transformer: BaseEstimator
+    ) -> torch.Tensor:
+        assert transformer.coerce_positive, "Beta distribution is only compatible with strictly positive data"
+        assert (
+            not transformer.log_scale
+        ), "Beta distribution is not compatible with log transformation - use LogNormal"
+        assert not transformer.center, "Beta distribution is not compatible with centered data"
+
+        scaled_mean = torch.sigmoid(parameters[..., 0] + target_scale[..., 0].unsqueeze(1))
```

Comment:
I wonder if it works even better if you take `torch.logit(target_scale[..., 0])` instead of `target_scale[..., 0]`.